File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-notifications_param.ts


import { ISearchNotifications_Param } from './isearch-notifications_param';

export class SearchNotifications_Param implements ISearchNotifications_Param {
  
	public SearchString: string = "";
	public AppUserID?: number;
	public ContextID?: number;
	public ContextTypeID?: number;
	public PageSize?: number;
	public PageNumber?: number;
	public SortOrder: string = "";
	public SortDirection: string = "";
  
}

File: GTTS/INL.SearchService/Models/SearchNotifications_Param.cs

namespace INL.SearchService.Models
{
    public class SearchNotifications_Param : ISearchNotifications_Param
    {
        public string SearchString { get; set; }
        public int? AppUserID { get; set; }
        public long? ContextID { get; set; }
        public int? ContextTypeID { get; set; }
        public int? PageSize { get; set; }
        public int? PageNumber { get; set; }
        public string SortOrder { get; set; }
        public string SortDirection { get; set; }
    }
}
